### PR TITLE
RENO-3088: Connect Slideshow to Drupal Data

### PR DESCRIPTION
**Pull request by isastettler**
[Jira Ticket](https://jira.nypl.org/browse/RENO-3088)

**This PR does the following:**
- Updates resolver with Slideshow section (sectionSix)
- Updates homepage query to include Slideshow section 
- Adds Slideshow component to homepage
- Creates a SlideshowCard component
- Removes unused code related to the Slideshow from Card component
- Updates image Type on CardItem and SlideshowItem 
- Updates `slide-show-card` variant styles
- Updates import of `SeeMore` type in component files
- Fixes test issue related to next/image on Slideshow and CardGrid tests

### Review Steps :
- [ ] Fetch new branches: `git fetch`
- [ ] Switch to relevant branch: `git checkout RENO-3088/connect-Slideshow-with-Drupal
- [ ] Run `npm install`
- [ ] Run all tests `npm test`
- [ ] All tests should pass and no error messages should be logged to terminal

### Front End Review Steps:
- [ ] Built site locally: `nm run build && npm start`
- [ ] Open [Homepage](http://localhost:3000/) in browser
- [ ] Compare section **New & Noteworthy** with [Live Homepage](http://nypl.org)

### CODE REVIEW

File: src/components/home-v1/Slideshow/SlideshowCard.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  id?: string;
+  title: string;
+  author: string;
+  genre: string;
+  audience: string;
+  image: ImageType;
+  url?: string;
+}
+
+interface CardProps {
```
Comment (nypl-wluisi) [inline src/components/home-v1/Slideshow/SlideshowCard.tsx:18]: Rename `CardProps` to `SlideshowCardProps`

File: src/components/home-v1/Slideshow/SlideshowCard.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { useStyleConfig } from "@chakra-ui/system";
+// Components
+import { Heading, Text, Link, Grid, GridItem, Box } from "@chakra-ui/react";
+import Image from "./../../shared/Image";
+import { ImageType } from "../../shared/Image/ImageTypes";
+
+export interface SlideshowCardItem {
+  id?: string;
```
Comment (nypl-wluisi) [inline src/components/home-v1/Slideshow/SlideshowCard.tsx:9]: I think id could be required? Do you remember why it was optional on the Card?

**Review by nypl-wluisi — CHANGES_REQUESTED** [review]
Looks good, just a few small changes.

File: src/components/home-v1/Slideshow/SlideshowCard.tsx (hunk 1 above)
Comment (isastettler) [reply src/components/home-v1/Slideshow/SlideshowCard.tsx:18]: Ugh good catch!

**Review by nypl-wluisi — APPROVED** [review]
Looks great!

**Pull request merged**